### DISCUSSION TITLE
Adobe Illustrator image colorspace is CMYK in old ImageMagick version but when upgrade it shows sRGB

### DISCUSSION THREAD

**Discussion by jetpack1116** [ImageMagick/ImageMagick#5740] [Help]
I have an Adobe Illustrator image.

When I use old Imagemagick ver 7.0.8-16,  then get its `colorspace`, the result is CMYK.

But when I upgraded to ImageMagick ver 7\.1.0-Q16, the resulting colorspace is sRGB.

I use the command:

```
magick identify -quiet -units PixelsPerInch -format {'image.colorSpace':'%[colorspace]'} C:\test\image\my_image.ai
```

Why is this?
Which is the correct colorspace?

If I open the file  in photoshop, it says colorspace is CMYK.
If I open the file in GIMP, it says colorspace is sRGB.

**Comment by urban-warrior** [comment]
Post a link to the image so we can inspect it.  We get expected results with the latest release of ImageMagick:

```
$ magick identify -quiet -units PixelsPerInch -format "{'image.colorSpace':'%[colorspace]'}" brookland-cover-as-CMYK.ai
{'image.colorSpace':'CMYK'}
$ magick identify -quiet -units PixelsPerInch -format "{'image.colorSpace':'%[colorspace]'}" brookland-cover-as-RGB.ai
{'image.colorSpace':'sRGB'}
```
